core/src/main/scala/kafka/server/BrokerLifecycleManager.scala

```diff
@@ -182,7 +182,7 @@ class BrokerLifecycleManager(
    */
   private[server] val eventQueue = new KafkaEventQueue(time,
     logContext,
-    threadNamePrefix.getOrElse(""),
+    threadNamePrefix.getOrElse("") + "BrokerLifecycleManager" + nodeId,
```

Comment:
I fired up this branch and yea `BrokerLifecycleManager1EventHandler` is not pretty :)

I'm not sure we need node ID in the thread name since we will presumably know which node a thread dump came from. It might actually be confusing since the convention we have is `{thread name}-{thread number in pool}` like "metrics-meter-tick-thread-1", "metrics-meter-tick-thread-2", "data-plane-kafka-request-handler-0", "data-plane-kafka-request-handler-1", etc. What about something like `broker-lifecycle-manager-event-handler` or `BrokerLifecycleManager-EventHandler`

I also noticed somewhere we are not prefixing the event queue thread name, there is just an `EventHandler` thread.